src/screens/Home/styles.tsx

```diff
@@ -1,8 +1,176 @@
 import { StyleSheet } from "react-native";
 
 export default StyleSheet.create({
-    profile: {
-        position: "absolute",
-        top: 20,
-    }
-})
\ No newline at end of file
+  separator: {
+    marginVertical: 30,
+    height: 1,
+    width: "90%",
+    backgroundColor: "#eee",
+  },
+
+  title1: {
```

Comment:
Pull from main and use the globalStyles text styling instead!!

Reply:
You may also get merge conflict in your yarn.lock, and if it's a lot just replace with the yarn.lock from main.

Reply:
Oh jk I see you're using the globalStyles styling, but it's just not the most recent version from main!